src/utils/auth.ts

```diff
@@ -13,19 +13,20 @@ export default class AuthUtils {
    *
    * @param keycloak keycloak instance
    */
-  public static keycloakInit = (keycloak: KeycloakInstance) => {
-    return new Promise<boolean>((resolve, reject) =>
-      keycloak.init({ onLoad: "login-required", checkLoginIframe: false })
+  public static keycloakInit = (keycloak: Keycloak) => {
```

Comment:
These helper functions can now be removed alltogether since Keycloak.init has a Promise-like return type. the Keycloak.init can just be used as awaited method wherever, no need for promise wrapper.